bin/fmt

```diff
@@ -1,14 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
 set -eu
 
 # go list will exclude the vendor directory.  We use grep to exclude generated
 # code.
-dirs=$(go list -f \{\{.Dir\}\} ./... | grep -v controller/gen)
+read -ra dirs <<< "$(go list -f \{\{.Dir\}\} ./... | grep -v controller/gen)"
 # go list will list all subdirectories and goimports acts recursively.  This
 # results in certain files being reported multiple time.  Therefore, we must
 # dedup them.
-files=$(bin/goimports -l "$dirs" | sort -u)
+files=$(bin/goimports -l "${dirs[@]}" | sort -u)
 
 if [ -n "$files" ]; then
```

Comment:
With the mapfile and arrays suggested, here we would have:

if [ ${#files[@]} -gt 0 ]; then
  for file in "${files[@]}"; do
    echo "$file"
    bin/goimports -d "$@" "$file"
  done
  exit 64
fi